tests/Propel/Tests/Generator/Behavior/Versionable/VersionableBehaviorObjectBuilderModifierTest.php

```diff
@@ -150,6 +150,29 @@ public static function setUpBeforeClass()
 EOF;
         QuickBuilder::buildSchema($schema4);
 
+    /**
+     *  Schema to test relation 1:1 versionable
+     */
+    $schema5 = <<<XML
+<database name="versionable_behavior_test_5">
+    <table name="VersionableBehaviorTestOneToOne">
+        <column name="id" type="integer" primaryKey="true" autoIncrement="true"/>
+        <column name="bar" type="varchar" size="32"/>
+        <behavior name="versionable" />
+    </table>
+
+    <table name="VersionableBehaviorTestOneToOneFK">
+        <column name="foo_id" type="integer" primaryKey="true"/>
+        <column name="bar" type="varchar" size="32"/>
+        <foreign-key foreignTable="VersionableBehaviorTestOneToOne">
+            <reference local="foo_id" foreign="id"/>
+        </foreign-key>
+        <behavior name="versionable" />
+    </table>
+</database>
+XML;
+        QuickBuilder::buildSchema($schema5);
```

Comment:
Only the build fails with 1:1? I don't see any real tests, only building of classes for some xml :)